charts/fleet/templates/job-migration.yaml

```diff
@@ -11,11 +11,16 @@ metadata:
   namespace: {{ .Release.Namespace }}
   annotations:
     {{- if .Values.mysql.enabled }}
-    {}
+    {{- with .Values.fleet.migrationJobAnnotations }}
```

Comment:
Instead of having this 3-line block inside the if and repeating for both cases, can we put it inside the annotations but outside of the if statement?  We'd only need it once that way.